### PR TITLE
Issue #1071 Resolution: Added new create-new-timetable button in dropdown menu

### LINKED ISSUE #1039: UX report
### Contact Details

_No response_

### Describe the Bug

Can you trade the icons for "new timetable" and "custom event"
Love the app a ton 🙏 overall I love the interface and how it looks. But I must've clicked on this "new timetable" thing at least 30 times every semester, which I don't think is meant to happen

### How can we reproduce the problem?

See image

### Expected Behavior

See image

### OS Version

None

### Which browsers are you seeing the problem on?

_No response_

### Additional Information

Capture d’écran 2023-10-03 à 19 48 21

### Code of Conduct

- [X] I agree to follow Semester.ly's Code of Conduct

### LINKED ISSUE #1071: Change location of "Create new timetable" action to the timetable drop-down menu
### Is your feature request related to a problem? Please describe.

Place to create a new timetable may be somewhat unintuitive

### Describe the solution you'd like.

It may be easier to put the "create new timetable" button in the timetable dropdown menu. So below seeing all of your timetable names, there would be an explicitly-labeled "Create new timetable" option (italicized or something, idk)

### Describe alternatives you've considered

Could maybe add this but also keep the "create new timetable" button as to not confuse old users? Not sure about this though since it may be redundant to have two buttons that serve the same purpose

### Additional Information

image

### Code of Conduct

- [X] I agree to follow Semester.ly's Code of Conduct

### PR DESCRIPTION
**Pull request by spencerckhuang**
for more intuitive ui/ux :)

- added new create-timetable button in dropdown
- removed create-timetable button above calendar

### CODE REVIEW

File: static/css/timetable/partials/side_bar.scss [hunk 1]
```diff
@@ -372,3 +372,11 @@
     }
   }
 }
+
+.create-new-timetable {
```
Comment (spencerckhuang) [inline static/css/timetable/partials/side_bar.scss:376]: added new css for 'add timetable button' in dropdown menu

File: static/js/redux/ui/Calendar.tsx [hunk 2]
```diff
@@ -272,21 +271,6 @@ const Calendar = (props: CalendarProps) => {
     />
   ) : null;
 
-  const addNewTimetableButton = (
```
Comment (spencerckhuang) [inline static/js/redux/ui/Calendar.tsx:275]: removed 'create new timetable' button from icon bar

File: static/js/redux/ui/CreateNewTimetableButton.tsx
Comment (spencerckhuang) [inline static/js/redux/ui/CreateNewTimetableButton.tsx:1]: component that is added to end of timetable dropdown menu

File: static/js/redux/ui/SideBar.tsx [hunk 3]
```diff
@@ -253,6 +254,7 @@ const SideBar = () => {
             <div className="tip" />
             <h4>{`${semester.name} ${semester.year}`}</h4>
             {savedTimetables}
+            <CreateNewTimetableButton setSidebarDropdown={setShowDropdown} />
```
Comment (spencerckhuang) [inline static/js/redux/ui/SideBar.tsx:257]: appending new component to end of dropdown list

File: static/js/redux/ui/containers/calendar_container.jsx
Comment (spencerckhuang) [inline static/js/redux/ui/containers/calendar_container.jsx:1]: removed line since Calendar.tsx no longer needs this func

**Comment by spencerckhuang** [comment]
**note: backend tests may have to be changed since some of them are looking for the icon i deleted...**

**Comment by spencerckhuang** [comment]
image
[Video demo](https://github.com/jhuopensource/semesterly/assets/112358919/97014700-e093-4c7e-a86a-0b2322c4792c)

**Review by JiaqiWang18 — APPROVED** [review]
Nice work

**Pull request merged**